src/web/components/Nav/ExpandedMenu/ExpandedMenuToggle/LabelButton.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+type Props = {
+    CHECKBOX_ID: string;
+    className: string;
+    ariaControls: string;
+    children: JSX.Element | JSX.Element[];
+};
+
+export const LabelButton = ({
```

Comment:
Is there a better name than 'LabelButton'? It sounds so innocent, which is risky because we probably don't want to invite future developers to use this elsewhere, given the fact it is built for a very specific purpose and contains so many overrides.

Reply:
`NavLabelButton` a better name?

Reply:
`NoJSButton`?